### ISSUE TITLE
请问变化检测中伪标签为什么不是通过ema_model获得的呢

### ISSUE THREAD

**Issue by Chelsea0216** [LiheYoung/UniMatch-V2#30]
您好，非常仰慕您的工作。
在查看变化检测中的伪标签获取的代码，发现是通过model而非ema_model得到的
with torch.no_grad():
                pred_u_w = model(imgA_u_w, imgB_u_w).detach()
                conf_u_w = pred_u_w.softmax(dim=1).max(dim=1)[0]
                mask_u_w = pred_u_w.argmax(dim=1)

和语义分割场景的代码不同：
with torch.no_grad():
                pred_u_w = model_ema(img_u_w).detach()
                conf_u_w = pred_u_w.softmax(dim=1).max(dim=1)[0]
                mask_u_w = pred_u_w.argmax(dim=1)
请问这样的设计是有意为之吗

**Comment by Chelsea0216** [comment]
看到论文里写的是为了加速。问题已解决
